android/app/build.gradle

```diff
@@ -45,7 +45,7 @@ android {
     defaultConfig {
         // TODO: Specify your own unique Application ID (https://developer.android.com/studio/build/application-id.html).
         applicationId "com.joinseeds.seedswallet"
-        minSdkVersion 18
+        minSdkVersion 21
```

Comment:
This is needed because otherwise we run into dex issues. 
64K max methods issue. 
This is due to firebase libs. 

Android 21+ is what we should be supporting anyways.